msprime/ancestry.py

```diff
@@ -249,7 +249,8 @@ def _parse_simulate(
     start_time=None,
     end_time=None,
     record_full_arg=False,
```

Comment:
Ah OK, let's not add any new features to ``_parse_simulate``. That's deprecated code now and we only add new features to ``sim_ancestry``.